src/main/java/de/azapps/kafkabackup/common/record/RecordJSONSerde.java

```diff
@@ -26,12 +27,19 @@
     private static final String TIMESTAMP_PROPERTY = "timestamp";
     private static final String KEY_PROPERTY = "key";
     private static final String VALUE_PROPERTY = "value";
+    private static final String HEADERS_PROPERTY = "headers";
+    private static final String HEADER_KEY_PROPERTY = "key";
+    private static final String HEADER_VALUE_PROPERTY = "value";
 
     public RecordJSONSerde() {
         this.mapper = new ObjectMapper();
         SimpleModule module = new SimpleModule();
-        module.addDeserializer(Record.class, new Deserializer());
-        module.addSerializer(Record.class, new Serializer());
+        module.addDeserializer(Record.class, new RecordDeserializer());
+        module.addDeserializer(ConnectHeaders.class, new HeadersDeserializer());
+        module.addDeserializer(ConnectHeader.class, new HeaderDeserializer());
+        module.addSerializer(Record.class, new RecordSerializer());
+        module.addSerializer(ConnectHeaders.class, new HeadersSerializer());
+        module.addSerializer(ConnectHeader.class, new HeaderSerializer());
```

Comment:
duplicated?

Reply:
my fault -  it's ok

Reply:
Yeah, the distinction between `headers` and `header` is really hard to see... But it comes from `ConnectHeaders` and `ConnectHeader`, so I don't know what to do really... 🤷‍♂️